index.html

```diff
@@ -46,5 +46,142 @@ <h4 class="welcome">LET'S CONNECT</h4>
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section>
+  <section class="card">
+    <figure class="picture">
+      <span id="third">-</span>
+    </figure>
+    <div class="works">
+      <h2>Tonic</h2>
+      <ul class="cover">
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section>
+  <section class="card">
+    <figure class="picture">
+      <span id="fourth">-</span>
+    </figure>
+    <div class="works">
+      <h2>Multi-Post Stories</h2>
+      <ul class="cover">
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section> 
+  <section class="about">
+    <article>
+      <h2 id="abh2">About<br>Myself</h2>
+      <p class="textab">Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+        contact me.</p>
+    </article>
+    <h4 id="footconect">LET'S CONNECT</h4>
+    <ul>
+      <li>
+        <a href="#">
+          <i class="fa fa-twitter"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fa fa-linkedin"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fa fa-github"></i>
+        </a>
+      </li>
+    </ul>
+    <a href="" class="prLnk">Get my resume</a>
+    <ul class="skills">
+      <li class="sklist">Langguages
+      <ul class="lngnames">
+        <li class="lngslid" id="js">JavaScript</li>
+        <li class="lngslid" id="hm">Html</li>
+        <li class="lngslid" id="cs">Css</li>
+      </ul>
+      </li>
+      <li class="sklist">Frameworks</li>
+      <li class="sklist">Skills</li>
+    </ul> 
+    <footer>
+      <article>
+        <h2 id="cont">Contact me</h2>
```

Comment:
- [ ] [OPTIONAL] I think your webpage will look better if you add the missing arrows and adjust the color of the icon links to make them match the original design. 
![image](https://user-images.githubusercontent.com/43698511/133910177-16ce8e4f-78fb-4c5e-a876-c6ad1034cc50.png)
![image](https://user-images.githubusercontent.com/43698511/133910186-7bca9229-0bf3-4546-beae-75cf38759813.png)